ocaml/idl/datamodel_common.ml

```diff
@@ -349,9 +349,9 @@ let get_product_releases in_product_since =
   in
   go_through_release_order release_order
 
-let next_release =
+let numbered_release version =
```

Comment:
I would have called this just "release"; there is no check that this is indeed numbered or using any particular format. Which could be done with a regular expression or Scanf.

Reply:
It's just to distinguish this case from the old releases, which have names.